docs/usage.rst

```diff
@@ -0,0 +1,249 @@
+.. code-block:: python
+
+    import edvart
+    from edvart.report_sections.dataset_overview import Overview
+
+    df = edvart.example_datasets.dataset_titanic()
+    report = edvart.Report(df)
+
+    report.add_overview(subsections=[
+        Overview.OverviewSubsection.QuickInfo,
+        Overview.OverviewSubsection.DataPreview,
+    ])
+
+
+.. _verbosity:
+
+Verbosity
+~~~~~~~~~
+
+A :py:class:`~edvart.report.Report` can be exported to a Jupyter notebook containing
+the code which generates the report. The code can be exported with different levels of detail,
+referred to as *verbosity*.
+
+It can be set on the level of the whole report or on the level of each
+section or subsection separately (see :ref:`sections-config`).
+
+The verbosity set on a lower level overrides the verbosity set on a higher level, i.e.
+the verbosity set on a subsection overrides the verbosity set on a section, which overrides
+the verbosity set on the report.
+
+EDVART supports three levels of verbosity:
+
+LOW
+   High level functions for whole sections are exported, i.e. each the output
+   of each section is generated by a single function call.
+   Suitable for small modifications such as changing parameters of the functions,
+   adding commentary to the report, adding visualizations which are not in EDVART, etc.
+
+MEDIUM
+   Same as low for report sections which do not consist of subsections.
+   For report sections which consists of subsections, each subsection is
+   exported to a separate function call.
+
+HIGH
+   The definitions of (almost) all functions are exported.
+   The functions can be modified and used as a starting point for custom analysis.
+
+
+Examples
+********
+
+.. code-block:: python
+
+    # Set default verbosity for all sections to Verbosity.MEDIUM
+    import edvart
+    from edvart import Verbosity
+
+    df = edvart.example_datasets.dataset_titanic()
+    edvart.DefaultReport(df, verbosity=Verbosity.MEDIUM).export_notebook("test-export.ipynb")
+
+
+.. code-block:: python
+
+    import edvart
+    from edvart import Verbosity
+
+
+    # Set report verbosity to Verbosity.MEDIUM but use verbosity Verbosity.HIGH for univariate analysis
+    df = edvart.example_datasets.dataset_titanic()
+    edvart.DefaultReport(
+        df,
+        verbosity=Verbosity.MEDIUM,
+        verbosity_univariate_analysis=Verbosity.HIGH,
+    ).export_notebook("exported-report.ipynb")
+
+
+Exporting Notebooks to HTML
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+An EDVART report :py:class:`~edvart.report.Report` can be directly exported
+to HTML via the :py:meth:`~edvart.report.ReportBase.export_html` method.
+
+To export a notebook to other formats including HTML, you may use a tool called
+`jupyter nbconvert` (https://nbconvert.readthedocs.io/en/latest/).
+
+For example, to export a notebook called `notebook.ipynb` using the `lab`
+template, you may use the following command:
+
+.. code-block:: bash
+
+   poetry run jupyter nbconvert --to html notebook.ipynb --template lab
```

Comment:
This is IMO to detailed. I would keep it at suggesting the tool. Otherwise if one only skims the documentation it might appear this command is the only way how to export to HTML.

Reply:
makes sense not to include how to use external tools

Reply:
removed